comictaggerlib/pagelisteditor.py

```diff
@@ -195,9 +231,42 @@ def item_move_event(self, s):
                 self.listOrderChanged.emit()
                 self.emit_front_cover_change()
                 self.modified.emit()
+        if s == "D" and self.chkDoublePage.isEnabled():
+            self.chkDoublePage.toggle()
+        if s == "M" and self.leBookmark.isEnabled():
+            self.leBookmark.setFocus()
+        if self.cbPageType.isEnabled():
+            idx = -1
+            if s == " ":
+                idx = self.cbPageType.findData("")
+            elif s == "A":
+                idx = self.cbPageType.findData(PageType.Advertisement)
+            elif s == "B":
+                idx = self.cbPageType.findData(PageType.BackCover)
+            elif s == "E":
+                idx = self.cbPageType.findData(PageType.Editorial)
+            elif s == "F":
+                idx = self.cbPageType.findData(PageType.FrontCover)
+            elif s == "I":
+                idx = self.cbPageType.findData(PageType.InnerCover)
+            elif s == "L":
+                idx = self.cbPageType.findData(PageType.Letters)
+            elif s == "O":
+                idx = self.cbPageType.findData(PageType.Other)
+            elif s == "P":
+                idx = self.cbPageType.findData(PageType.Preview)
+            elif s == "R":
+                idx = self.cbPageType.findData(PageType.Roundup)
+            elif s == "S":
+                idx = self.cbPageType.findData(PageType.Story)
+            elif s == "X":
+                idx = self.cbPageType.findData(PageType.Deleted)
+            if idx > -1:
+                self.cbPageType.setCurrentIndex(idx)
+                self.change_page_type(idx)
```

Comment:
Same as above using qactions will be the best option

Reply:
Using QActions now